actions/workflows/delete_policy.meta.yaml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Naming thing here - the workflow definition itself (this file) can technically be called anything. Often it has the same filename as the workflow metadata (the file in the actions directory). But if you do have different names for the metadata file and the workflow definition, they should be the other way around. E.g. in your case you would have `actions/delete_policy.meta.yaml` and then `actions/workflows/delete_policy.yaml`